### ISSUE TITLE
Wrong Warning: contextMenus.create is not supported

### ISSUE THREAD

**Issue by erosman** [mozilla/amo-validator#554]
Example:
```
https://addons.mozilla.org/en-US/developers/addon/invert-colors/file/670527/validation
https://addons.mozilla.org/en-US/firefox/files/browse/670527/file/background.js#L59
```

> contextMenus.create is not supported
> This API has not been implemented by Firefox.
> background.js line 59
> 
> contextMenus.onClicked is not supported
> This API has not been implemented by Firefox.
> background.js line 66

```
browser.contextMenus.create({
  id: "InvertColors",
  title: "Invert Colors"
});

browser.contextMenus.onClicked.addListener(toggleColors);
```

More:
`https://addons.mozilla.org/en-US/firefox/files/compare/670891...291288/file/main.js#top`

`var id = chrome.contextMenus.create({ ...`

**Comment by wagnerand** [comment]
Thanks for the report. 
This repository is for validation of legacy add-ons only. Please file an issue at https://github.com/mozilla/addons.
